packages/apps/graph-client/src/components/chainweb/chain-block.tsx

```diff
@@ -0,0 +1,94 @@
+import { Block } from '../../__generated__/sdk';
+import { keyframes, styled } from '../../styles/stitches.config';
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { TimeTicker } from './time-ticker';
+
+import { RocketIcon, TimerIcon } from '@radix-ui/react-icons';
+import React from 'react';
+
+const Container: any = styled('div', {
+  borderRadius: '$md',
+  position: 'relative',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  m: '$1',
+
+  // NOTE: Needs to be updated to be responsive
+  height: '$blockWidth',
+  width: '$blockWidth',
+  border: '1px solid $mauve7',
+});
+
+const BlockEntrance: any = keyframes({
+  '0%': {
+    transform: 'scale(0)',
+    animationTimingFunction: 'ease-in',
+  },
+  '70%': {
+    transform: 'scale(1.1)',
+    animationTimingFunction: 'ease-in',
+  },
+  '100%': {
+    transform: 'scale(1)',
+    animationTimingFunction: 'ease-out',
+  },
+});
+
+const Content: any = styled('div', {
+  zIndex: 2,
+  animation: `${BlockEntrance} 0.4s`,
+  height: '$blockWidth',
+  width: '$blockWidth',
+  background: '$$color',
+  borderRadius: '$md',
+  display: 'flex',
+  alignItems: 'center',
+  px: '$2',
+  py: '$1',
+});
+
+interface IChainBlockProps {
+  color: string;
+  block?: Block;
+  textColor: string;
+}
+
+export function ChainBlock(props: IChainBlockProps): JSX.Element {
+  const { color, textColor, block } = props;
+
```

Comment:
early return

if (!block) return null

Reply:
The `Container` component that is still rendered when there is no block data also applies styling so we wouldn't want to return null in this instance

Reply:
a yes , missed that

Reply:
a yes , missed that